app/src/main/java/to/dev/dev_android/view/main/view/MainActivity.kt

```diff
@@ -35,13 +36,21 @@ class MainActivity : BaseActivity<ActivityMainBinding>(), CustomWebChromeClient.
         setWebViewSettings()
         savedInstanceState?.let { restoreState(it) } ?: navigateToHome()
         handleIntent(intent)
-        PushNotifications.start(getApplicationContext(), BuildConfig.pusherInstanceId);
-        PushNotifications.addDeviceInterest("broadcast");
+        PushNotifications.start(applicationContext, BuildConfig.pusherInstanceId)
+        PushNotifications.addDeviceInterest("broadcast")
     }
 
     override fun onResume() {
         if (intent.extras != null && intent.extras["url"] != null) {
-            binding.webView.loadUrl(intent.extras["url"].toString())
+            val targetUrl = intent.extras["url"].toString()
+            try {
+                val targetHost = Uri.parse(targetUrl).host ?: ""
+                if (targetHost.contains(BuildConfig.baseHostname)) {
+                    binding.webView.loadUrl(targetUrl)
+                }
+            } catch (e: Exception) {
+                Log.e(LOG_TAG, e.message)
```

Comment:
Is this log uploaded anywhere or just kept locally?

Reply:
Only logged locally at the moment. A 3rd party integration for crash reporting and/or error logging is definitely in my radar for the future 🙂